templates/content/GeoBlazor.Template.Maui/Pages/Index.razor

```diff
@@ -0,0 +1,16 @@
+﻿@page "/"
+
+<h1>Hello, world!</h1>
+
+Welcome to your new app.
+
+<MapView Longitude="_longitude" Latitude="_latitude" Zoom="11" Style="height: 400px; width: 400px;">
+    <WebMap>
+        <PortalItem Id="4a6cb60ebbe3483a805999d481c2daa5" />
```

Comment:
I cant remember if you need an api key to fire these maps up the first time...

Reply:
you do, the readme.md contains where to set the apikey